django_browserid/tests/test_views.py

```diff
@@ -234,4 +234,4 @@ def test_django_csrf(self):
     def test_never_cache(self):
         request = self.factory.get('/browserid/csrf/')
         response = self.view.get(request)
-        self.assertEqual(response['Cache-Control'], 'max-age=0')
+        self.assertTrue('max-age=0' in response['Cache-Control'])
```

Comment:
I can't find the log line anymore but IIRC it was stuff like `no-cache` and `must-revalidate`. I figured that just checking for this much was good enough for our test.